Source/AtomicWebView/WebTexture2D.cpp

```diff
@@ -392,6 +390,14 @@ void WebTexture2D::SetSize(int width, int height)
 
 #endif
 
+#if defined(ATOMIC_OPENGL) && defined(ATOMIC_PLATFORM_WINDOWS) 
+    //Force width and height to multiples of 2 so that we don't squish pixels
+    //This doesn't seem to make the text completely sharp but it makes it usable for now.
+    width = (width | 2) + 1;
+    height = (height | 2) + 1;
+    texture_->SetFilterMode(TextureFilterMode::FILTER_NEAREST);
+#endif
```

Comment:
Ok, this needs further looking into as having a pixel discrepancy between requested size and what the size of the texture actually is, only on Windows, and only when under GL, could cause some hard to track down memory buffer access violations.  It also breaks the early out at the top of the method for size == textureSize when requesting odd sized width/height, so will always will recreate